sql/transform/node_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2020-2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transform
+
+import (
+	"testing"
+
+	"github.com/dolthub/go-mysql-server/sql"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTransformUp(t *testing.T) {
```

Comment:
Could use a few more tests of the various transform methods

Fine to do in a followup, not like we've ever had them before